src/components/GovBanner/GovBanner.tsx

```diff
@@ -154,7 +154,8 @@ export const GovBanner = ({
               <img
                 className="usa-banner__icon usa-media-block__img"
                 src={dotGovIcon}
-                alt="Dot gov"
+                alt=""
+                aria-hidden="true"
```

Comment:
Oh yea, I think I didn't add these because of https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-redundant-roles.md

Weirdly enough, eslint doesn't complain when I add `role="img"` to these `img`s, but if I add it to the lock icon `img` tag it complains.

Reply:
For now its updated in a9ea0d4

We still want to use the `img` tag for our implementation of the lock icon over the inline `svg` that USWDS is using, correct?

Reply:
weird! I think I ran into a similar eslint a11y conflict with USWDS markup in the Date Picker and I erred in favor of USWDS markup.

yes I wouldn't worry too much about using the inline `svg` vs `img`